libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -826,6 +1011,80 @@ void AP_CRSF_Telem::process_param_write_frame(ParameterSettingsWriteFrame* write
 #endif
 }
 
+// get status text data
+void AP_CRSF_Telem::calc_status_text()
+{
+    if (!_statustext.available) {
+        WITH_SEMAPHORE(_statustext.sem);
+        // check link speed
+        if (_telem_rf_mode != AP_RCProtocol_CRSF::CRSF_RF_MODE_150HZ) {
```

Comment:
Perhaps a `bool high_speed_telemetry() const` call might be useful as we do this check in multiple places.

Reply:
agree, done